### ISSUE TITLE
audit-avt1-search: "searchbox" has no label

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1701]
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Components - Search: Usage & Code](https://www.carbondesignsystem.com/components/search/usage)

## Detailed Description: 

Form control with "searchbox" role has no associated label

<img width="1680" alt="Screen Shot 2020-09-10 at 3 11 42 PM" src="https://user-images.githubusercontent.com/32720851/92789338-c0883b00-f378-11ea-9a6d-417be3d455b1.png">

## What to do: 
Add a for attribute to a visible label with the id of this control (e.g. <label for="controlID">);

## Why is this important?
Associating a meaningful label with every UI control allows the browser and assistive technology to expose and announce the control to a user.
